### PR TITLE
Fix bug

### PR DESCRIPTION
**Pull request by tianyu-z**

### CODE REVIEW

**Comment by tianyu-z** [comment]
sys at line 17 note defined

**Comment by kennymckormick** [comment]
Hi, @tianyu-z , 
Do you actually meet some bug with paligemma? I think the statement `from ..smp import *` actually also import the `sys` lib. In `smp`, we import a lot of frequently used libs.

**Comment by tianyu-z** [comment]
Yes, when I run codes without explicitly import sys, it raises a bug.

**Comment by tianyu-z** [comment]
I have just switched off my PC. If needed, I can send a screenshot after I
get up.

**Comment by tianyu-z** [comment]
image
Here is the screenshot of the bug.

**Comment by tianyu-z** [comment]
Additionally, when I run evaluations for `omnilmm`. It also has some problem about the module `omnilmm` cannot be found issue.

**Comment by junming-yang** [comment]
> Additionally, when I run evaluations for `omnilmm`. It also has some problem about the module `omnilmm` cannot be found issue.

Do you set the `OmniLMM_ROOT` in the config.py?

**Comment by junming-yang** [comment]
> image
> Here is the screenshot of the bug.

Thanks for your response. Maybe we can `import sys` in the vlmeval/smp/vlm.py file. Can you update this PR?

**Comment by tianyu-z** [comment]
> Do you set the OmniLMM_ROOT in the config.py?

Oh, I didn't find an instruction for it. If this is required, it would be great to have some instruction somewhere.

I recommend we could have a model card for every submitted model.
e.g. How to run it? What are the dependencies (especially the version of the transformers)? Models such as `omnilmm`, `DocOwl2`, `instructblip_7b` need more instruction to run the evaluation. Otherwise, as time goes by, more and more models will fail to be evaluated easily.

> Thanks for your response. Maybe we can import sys in the vlmeval/smp/vlm.py file. Can you update this PR?
Done

Moreover, I didn't manage to evaluate `instructblip_7b` even if I followed the instruction of their [original github](https://github.com/salesforce/lavis) to build up the env and requirements. Do you know the solution? Thanks!
image

**Comment by junming-yang** [comment]
> > Do you set the OmniLMM_ROOT in the config.py?
> 
> Oh, I didn't find an instruction for it. If this is required, it would be great to have some instruction somewhere.
> 
> I recommend we could have a model card for every submitted model. e.g. How to run it? What are the dependencies (especially the version of the transformers)? Models such as `omnilmm`, `DocOwl2`, `instructblip_7b` need more instruction to run the evaluation. Otherwise, as time goes by, more and more models will fail to be evaluated easily.
> 
> > Thanks for your response. Maybe we can import sys in the vlmeval/smp/vlm.py file. Can you update this PR?
> > Done
> 
> Moreover, I didn't manage to evaluate `instructblip_7b` even if I followed the instruction of their [original github](https://github.com/salesforce/lavis) to build up the env and requirements. Do you know the solution? Thanks! image

OmniLMM should be installed first, following the official repo instruction. If you want to use OmniLMM in VLMEvalKit, you need to set the repo dir to `OmniLMM_ROOT` in config.py line 9.

Some models require additional installation. We marked them in the readme `Supported PyTorch / HF Models` card with the notation. Additionally, the special instruction is in the quickstart.md `Step 1. Configuration` part. For most installation-required models,  you just need to clone the official repo and follow their instructions. Only InstructBLIP, LLaVA-v1-7B, MiniGPT-4 and PandaGPT-13B models need to modify the config files in `vlmeval/vlm/misc` to configure LLM path and ckpt path.

The recommended version of the transformers is also listed in the readme. 

instructblip_7b needs to set the `llm_model` to define the path to the Vicuna checkpoint in `vlmeval/vlm/misc/blip2_instruct_vicuna7b.yaml`

File: vlmeval/vlm/paligemma.py
Comment (junming-yang) [inline vlmeval/vlm/paligemma.py:1]: Do not need to re-import sys.  Will from ..smp import *

**Pull request merged**